script/Ciel/Day 13/13_01_02_CIEL13_6.txt

```diff
@@ -494,128 +605,147 @@ The fact of the matter is that I'm awfully calm right now.
+I cast aside the last shreds of compassion and sympathy I still had.
 }
 [sha:05d75bcd3c10c83db1e1033e60dff61f002af1e0]{
 -- Page 147, Offset 35666.
 -- 「訊いた俺が馬鹿だった―――！」
-"I was stupid to have asked that―――!"
+"I shouldn't have bothered to ask―――!"
 }
 [sha:3d6f9f7ff80bbed850681a47a144f6a9341d6e3a]{
 -- Page 147, Offset 35667.
 -- こめかみの頭痛を噛み潰して、二度目の接触を試みた。
-Bearing with the pain in my temples by gritting my teeth, I attempt to make contact for the second time.
+Gritting my teeth to distract from the pain in my temple,
+ I attempt to make contact for the second time.
 }
 [sha:064fafe68c4d7a82d8f4cf09154e58387450929b]{
 -- Page 148, Offset 35668.
 -- 距離にして<三間|６メートル>。踏み込みは二足で事足りる。
+// Daddy long legs with the 3m stride
 She's six meters away. I should be able to close the distance in two steps.
 }
 [sha:c71b5800fd846a81e8b32e3d65886897e147a00f]{
 -- Page 148, Offset 35669.
 -- 一足。こちらを迎え撃とうと構えるノエルの右手と、地面を踏みしめる右足のつま先を確認する。
-The first step. I check Noel's right hand, which she intends on using to intercept me, and the tip of her right foot, which is firmly planted on the ground.
+One step.
+ I glance at Noel's right hand, which is poised to intercept me,
+ and the tip of her right foot, which is firmly planted on the ground.
 }
 [sha:c778f65aee2d351f92e94377bfa556a666f0e063]{
 -- Page 148, Offset 35670.
 -- 右手に構えた武器を突き出すのなら軸足は左になる。右足に力が入るという事は、
-If she's thrusting her weapon with her right hand, then her left foot would act as the pivot. But if she puts her strength into her right foot...
+If she was going to thrust with her right hand, she'd want to use her left foot as a pivot.
+ But if she's using her right foot instead, then that means...
 }
 [sha:4caf6f044b12c5636d52c13bf90e4b80dd0c666e]{
 -- Page 149, Offset 35671.
 -- 二足。敵の間合いに入った。
-The second step. I've entered within range of the enemy.
+Two steps.
+ I've entered within range of the enemy.
 }
 [sha:0b9637d3b0454d3028a2b165cb20bbee9c14da67]{
 -- Page 149, Offset 35672.
 -- 武器の<射程|リーチ>でこちらは劣っている。
-My weapon's reach is inferior compared to hers.
+My weapon's reach is inferior to hers.
 }
 [sha:9cf1a745da91ed8e7d3f04a7cf74f12fd2cce3ae]{
 -- Page 149, Offset 35673.
 -- まずはこの一撃をかいくぐらなければ<蠢|うごめ>く線を切断できない。
-For starters, if I don't slip past her first attack, then I won't be able to sever that squriming line.
+If I can't slip past her opening attack,
+ I won't get a chance to sever that squirming line.
 }
 [sha:b1111a067246a3c05eba990952970a2b6e2bfc6f]{
 -- Page 150, Offset 35674.
 -- 俺は直前で急停止して、心臓めがけて突き出された凶器を躱して今度こそ、
-I stop immediately before she thrusts her weapon at my heart, evading her attack, and then―――
+I'll stop just short of her,
+ dodge the skewer aimed at my heart,
+ and then―――
 }
 [sha:9a41d9278a98110777bfd56e338ad1f05bf27db9]{
 -- Page 150, Offset 35675.
 -- 「なーんて、<真|マ><剣|ジ>で付き合ってあげると思っちゃった？」
-"Yeah right. Did you really think that I would go along with that?"
+"Yeah right.
+ Did you seriously think I'd let that happen?"
```

Comment:
She's using her sweet voice again.
>%{i}Juuust%{/i} kidding.
>Did you really think I'd let that happen?

Maybe add a heart to the first one idk, lol.